Helper/Webhook/OrderClosedWebhookHandler.php

```diff
@@ -38,6 +39,11 @@ class OrderClosedWebhookHandler implements WebhookHandlerInterface
     /** @var OrderPaymentCollectionFactory */
     private $adyenOrderPaymentCollectionFactory;
 
+    /**
+     * @var SerializerInterface
+     */
+    private $serializer;
```

Comment:
Could you please define the required serializer (Magento\Framework\Serialize\Serializer\Json) explicitly in `di.xml` for this class? Otherwise, it might use PHP serializer instead of JSON serializer. I assume JSON serializer is required here. [Example here](https://github.com/Adyen/adyen-magento2/blob/main/etc/di.xml#L1637)